Source/Plugins/EditorModules/ObjectInspector/ObjectInspectorPlugin.cs

```diff
@@ -57,25 +51,24 @@ protected override IDockContent DeserializeDockContent(Type dockContentType)
 			this.isLoading = false;
 			return result;
 		}
-		protected override void SaveUserData(XElement node)
+		protected override void SaveUserData(PluginSettings pluginSettings)
 		{
-			for (int i = 0; i < this.objViews.Count; i++)
+			foreach (ObjectInspector objectInspector in this.objViews)
 			{
-				XElement elem = new XElement("ObjectInspector");
-				elem.SetAttributeValue("id", i);
-				node.Add(elem);
-				this.objViews[i].SaveUserData(elem);
+				objectInspector.SaveGridExpandState();
 			}
+			pluginSettings.Set(this.userSettings);
 		}
-		protected override void LoadUserData(XElement node)
+
```

Comment:
Style: Remove the newline here to keep the current member grouping in place, or (if changing the group layout) keep together `SaveUserData` and `LoadUserData` in the same group.